build/linux/build_linux_iwyu.sh

```diff
@@ -2,6 +2,9 @@
 # Copyright (c) Microsoft. All rights reserved.
 # Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+#re-eable with https://msazure.visualstudio.com/One/_workitems/edit/29593386
+exit 0
```

Comment:
I'd do the disable by commenting the yml portion out. That is what we usually do and that would be the first place to look. This would likely come as a surprise after wondering how come it returned 0 but did nothing ...
/consider

Reply:
Moved to yml

Reply:
As a side note, same yml, a tiny wee above: